packages/typespec-client-generator-core/test/types.test.ts

```diff
@@ -2120,6 +2120,78 @@ describe("typespec-client-generator-core: types", () => {
       });
     });
 
+    it("multipart with encoding as binary bytes should not raise error", async function () {
+      const diagnostics = await runner.diagnose(
+        `
+        @service({title: "Test Service"}) namespace TestService;
+        model EncodedBytesMFD {
+          @encode("binary")
+          pictures: bytes;
+        }
+        
+        @put op multipartOp(@header contentType: "multipart/form-data", @body body: EncodedBytesMFD): void;
+        `
+      );
+      getAllModels(runner.context);
+      expectDiagnostics(diagnostics, {
+        code: "@azure-tools/typespec-client-generator-core/encoding-multipart-bytes",
+      });
+    });
+
+    it("multipart with bytes projectedName should not raise error", async function () {
+      const diagnostics = await runner.diagnose(
+        `
+        @service({title: "Test Service"}) namespace TestService;
+        model EncodedBytesMFD {
+          @projectedName("json", "pics")
+          pictures: bytes;
```

Comment:
This is failing which I don't think we should ? @iscai-msft @lmazuel @joheredi

Reply:
I think this test doesn't make sense it says it shouldn't raise an erorr but you expect one line 2154

Reply:
Oh sorry, I was trying to repro a case I saw.